main/webapp/modules/core/langs/translation-en.json

```diff
@@ -577,7 +579,8 @@
     "core-views/word-facet": "Word facet",
     "core-views/add-id-column": "Add entity identifiers column…",
     "core-views/add-id-column2": "Add a column containing entity identifiers from items",
-    "core-views/add-id-col": "Add column containing entity identifiers on \"$1\"",
+    "core-views/add-id-col": "Add column containing entity identifiers on $1",
```

Comment:
Here you have removed the double quotes around the column name, but I would keep them (to separate the user-provided text from the system text more cleanly). This should help get the Cypress test pass again.